server/core/templates/credentials_update_partial.html

```diff
@@ -91,21 +91,27 @@
         (% let primary_state = primary_state %)
         (% include "credentials_update_primary.html" %)
 
-        <hr class="my-4"/>
+        <div class="my-5"></div>
+        <div class="my-5"></div>
 
-        <div class="d-flex flex-row flex-wrap justify-content-between">
-            <button class="btn btn-danger btn-lg" hx-post="/ui/api/cu_cancel" hx-target="body">Cancel</button>
-            <span class="d-inline-block" tabindex="0" data-bs-toggle="tooltip" data-bs-title="Resolve the warnings at the top.">
-                <button
-                        class="btn btn-success btn-lg"
-                        type="submit"
-                        hx-post="/ui/api/cu_commit"
-                        hx-boost="false"
-                        (% if !warnings.is_empty() %)
-                        disabled
-                        (% endif %)
-                >Submit Changes</button>
-            </span>
+        <div id="cred-update-commit-bar">
```

Comment:
bootstrap has an alert component I think it'll be better if you use that. I don't know if we have any ambitions to make kanidm look amazing (looking at you @Firstyear) but it'll be nicer if everything looks part of something. Also the point of using something like Bootstrap is to minimize custom CSS and they become quite unmaintainable pretty quickly

Reply:
I'd definitely request [bootstrap alerts](https://getbootstrap.com/docs/5.3/components/alerts/) for this please. You should be able to get it to float over with a z-index if that's your jam.

Reply:
Actually, [Toasts](https://getbootstrap.com/docs/5.3/components/toasts/) are another option which might be more effective/dynamic.

Reply:
![image](https://github.com/user-attachments/assets/0985c200-de35-47c3-899b-8234f75b9306)
Like this ?

Reply:
> bootstrap has an alert component I think it'll be better if you use that. I don't know if we have any ambitions to make kanidm look amazing (looking at you @Firstyear) 

I'm not skilled enough as a web dev to make it look nice - i'll defer to you and others who actually have those skills :) 

My background is all in server side stuff, so I'll trust your decisions :)